torch/lib/c10d/ProcessGroup.hpp

```diff
@@ -162,6 +162,22 @@ class ProcessGroup {
       std::vector<std::vector<at::Tensor>>& inputTensors,
       const ReduceScatterOptions& opts = ReduceScatterOptions()) = 0;
 
+  virtual std::shared_ptr<ProcessGroup::Work> alltoall_base(
+      std::vector<at::Tensor>& outputTensors,
+      std::vector<at::Tensor>& inputTensors,
+      std::vector<int>& outputSplitSizes,
+      std::vector<int>& inputSplitSizes,
+      const AllToAllOptions& opts = AllToAllOptions()) {
+      throw std::runtime_error("ProcessGroup does not support alltoall");
+  }
```

Comment:
So, should I change this to following?
```suggestion
  virtual std::shared_ptr<ProcessGroup::Work> alltoall_base(
      at::Tensor& outputTensor,
      at::Tensor& inputTensor,
      std::vector<int64_t>& outputSplitSizes,
      std::vector<int64_t>& inputSplitSizes,
      const AllToAllOptions& opts = AllToAllOptions()) {
      throw std::runtime_error("ProcessGroup does not support alltoall");
  }
```

Reply:
this seems good. thank you!